app/views/journals/index.html.slim

```diff
@@ -6,24 +6,18 @@ h1= t_title(:index, Journal)
           tr
             th= t_attr(:created_at)
             th= t_attr(:category)
-            th= t_attr(:title)
             th= t_attr(:text)
             th= t_attr(:author)
-            th colspan=3
+            th
         tbody
           - @journaled.journals.each do |record|
             tr
-              td= l(record.created_at)
+              td= l(record.created_at.to_date)
               td= t("category.#{record.category}")
-              td= record.subject
-              td= record.body&.truncate(40)
+              td= word_wrap(record.body)
               td= link_to record.user.full_name, record.user.profile
-              td= link_to t_action(:show), url_for([@journaled, record])
               td= link_to t_action(:edit), url_for([@journaled, record]) + '/edit'
-              td= link_to t_action(:destroy), url_for([@journaled, record]), confirm_deleting(record)
 .row
-  .col-xs-12
-= form_navigation_btn :new
+  .col-xs-12= form_navigation_btn :new
```

Comment:
`form_navigation_btn` allready creates the `div.row > div.col-xs-12`, so you should use `= form_navigation_btn :new` without them.

https://github.com/panter/aoz-003/blob/develop/app/helpers/application_helper.rb#L25